src/trust/types.ts

```diff
@@ -18,14 +18,29 @@ const RelyingPartyMetadata = z.object({
 // instruct the Wallet Solution on how to render the credential correctly
 type CredentialDisplayMetadata = z.infer<typeof CredentialDisplayMetadata>;
 const CredentialDisplayMetadata = z.object({
+  name: z.string(),
+  locale: z.string(),
+  logo: z
+    .object({
+      url: z.string(),
+      alt_text: z.string(),
+    })
+    .optional(), //should not be optional
+  background_color: z.string().optional(), // should not be optional
+  text_color: z.string().optional(), // should not be optional
```

Comment:
Add a TODO with the related task. But if they are not used within the entire codebase we can eliminate them completely and leave only what is needed in the type.

Reply:
Done in b8604c698f10fe6739920a8104c80bb3e069cf01, I'd rather keep them optional at the moment but we can decide later if that makes sense to you. Thanks.